### PR TITLE
Update docs building

### PR DESCRIPTION
**Pull request by juhasch**
* Simplify docs generation (only `make html` required)
* Prepare for readthedocs
* Remove `extensions.rst`. Will be built by `md2rst.py`

### CODE REVIEW

File: docs/make.bat [hunk 1]
```diff
@@ -46,15 +45,14 @@ if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	for /d %%i in (source\*) do rmdir /q /s %%i
-
+	del /q /s source\extensions.rst
```
Comment (jcb91) [inline docs/make.bat]: could we maybe use the full package name of `jupyter_contrib_nbextensions` here too? Or at least specify `nbextensions` over `extensions`

Comment (juhasch) [inline docs/make.bat]: Do you think the name matters ?
It is just a filename that is being generated and contains the list of notebook extensions.

Comment (jcb91) [inline docs/make.bat]: No, it doesn't matter, in the sense that it'll still work fine regardless. But, it's just slightly less surprising/confusing to keep all the names consistent

**Comment by juhasch** [comment]
@jcb91 Readthedocs does not seem to know Pandoc, so it will not build HTML docs:
```
 conda env create --name latest --file /home/docs/checkouts/readthedocs.org/user_builds/jupyter-contrib-nbextensions/checkouts/latest/docs/environment.yml

Fetching package metadata: ....
Solving package specifications: .
Error:  Package missing in current linux-64 channels: 
  - pandoc
```
 Do you have a hint for me how to proceed ?

**Comment by jcb91** [comment]
@juhasch apologies for the delayed reply. I've not built anything on rtd before, so I'm not really in a position to offer any advice, I'm afraid. Perhaps we could do this not through conda? Or wait for the bugfix :smile:

**Comment by juhasch** [comment]
Sure, I will investigate further. I guess your suggestion not to use conda for now is what I will try next.

**Pull request merged**